src/components/FAQSideNav.astro

```diff
@@ -0,0 +1,44 @@
+---
+const { headings, title } = Astro.props;
+---
+
+<div class="usa-layout-docs__sidenav desktop:grid-col-4">
+  <nav class="sp_site_nav" aria-label="Secondary navigation">
+    <h4>{title}</h4>
+    {
+      headings.map((h) => (
+        <ul class="usa-sidenav">
+          <li class="usa-sidenav__item">
+            <a
+              href={`#${h.slug}`}
+              onclick={`document.querySelector('#${h.slug}').scrollIntoView({behavior: "smooth"}); event.preventDefault();`}
+            >
+              {h.text}
+            </a>
+            {h.submenu.length > 0 && (
+              <ul class="usa-sidenav__sublist">
+                {h.submenu.map((h3) => (
+                  <li class="usa-sidenav__item">
+                    <a
+                      href={`#${h3.slug}`}
+                      onclick={`document.querySelector('#${h3.slug}').scrollIntoView({behavior: "smooth"}); event.preventDefault();`}
+                    >
+                      {h3.text}
+                    </a>
+                  </li>
+                ))}
+              </ul>
+            )}
+          </li>
+        </ul>
+      ))
+    }
+  </nav>
+</div>
+<style scoped>
+
+.usa-sidenav a:not(:hover):not(.usa-current) {
+  background-color: #ecf1f7;
```

Comment:
This is a nice solution to the problem. I think, though, that this should probably be added to the sass file  in `sass/_components/side_nav.scss` instead of hard-coding the color here. I know it's only for this one page, but if the value is hard coded here, we can't change this color by changing the theme variable. 

You might be able to do something like this, then gulp compile it. Then it's guaranteed to be the same color as everything else even if things change:
```
.usa-sidenav {
    a{
        &:not(:hover){
            &:not(.usa-current){
                background-color:color('base-lightest');
            }
        }
    }
```

Reply:
I set it as page level is because I thought it would be safer to be inline, I thought it may mess up other pages using same class, but after I move it to side_nav.scss, seems about page is still working fine, no conflicts. So inline style are removed and style are in side_nav now.